spec/system/admin/order_spec.rb

```diff
@@ -409,14 +409,7 @@ def new_order_with_distribution(distributor, order_cycle)
       fill_in "order_bill_address_attributes_city", with: "Kansas"
       fill_in "order_bill_address_attributes_zipcode", with: "SP1 M11"
 
-      # The country is already selected and we avoid re-selecting it here
-      # because it would trigger an API call which we would need to wait for
-      # while we have no visual indicator for the wait.
-      #
-      # Ideally, we would implement a visual cue like disable the state field
-      # while the data is fetched from the API.
-      #
-      # select "Australia", from: "order_bill_address_attributes_country_id"
+      select "Australia", from: "order_bill_address_attributes_country_id"
```

Comment:
It looks like this has been problematic in the past, so hopefully this doesn't bring back an old problem 🤞